tsd/src/tsd/CMakeLists.txt

```diff
@@ -77,6 +81,12 @@ PRIVATE
   $<BUILD_INTERFACE:tsd_tinyexr>
   $<BUILD_INTERFACE:tsd_tiny_obj_loader>
   $<BUILD_INTERFACE:tsd_tinyply>
+  $<$<BOOL:${USE_NEURAL_GRAPHICS_PRIMITIVES}>:torch>
+)
+
+project_compile_definitions(
```

Comment:
I'm not asking you to make any of these changes, but FYI CMake is additive with the target_ (and my own project_) commands. So you could instead do something like:

```cmake
if (TSD_USE_TORCH)
  find_package(Torch REQUIRED)
  project_link_libraries(PRIVATE ${TORCH_LIBRARIES})
  project_compile_definitions(PRIVATE USE_NEURAL_GRAPHICS_PRIMITIVES) # or better yet 'USE_TORCH'
endif()
```

You can see this pattern with other external library dependencies used for importing.